umh-core/pkg/fsmv2/cpu/cpu_admission_deadline_warning_test.go

```
@@ -0,0 +1,312 @@
+			Expect(spy.sentryWarnMsgs[0]).To(Equal("cpu_admission_deadline_never_measured_signal"),
+				"the event name is a fixed grouping key, never instance-varying")
+
+			// (e) The varying data rides in the structured fields — never in the
+			// message prose. sentry/hook.go promotes only a fixed tag set; these
+			// fields land in Contexts["umh_context"], which the Sentry issue page
+			// shows per event but does not index for search (sentry/doc.go). What
+			// keeps the occurrences together is the fixed event name asserted in
+			// (c); the fields are what an operator reads once inside that one
+			// issue, and what the structured log line carries locally. A
+			// regression that stops emitting the structured fields must fail here.
+			Expect(len(spy.sentryWarns)).To(Equal(1), "the warning is recorded once, like the call")
```

golangci-lint (2.12.2)

[error] ginkgo-linter: wrong length assertion. Consider using `Expect(spy.sentryWarns).To(HaveLen(1), "the warning is recorded once, like the call")` instead

(ginkgolinter)